js/data.json

```diff
@@ -986,6 +986,12 @@ osts = [
 		"link" : "http://66.90.93.122/ost/naruto-shippuden-original-soundtrack-1/huxtigyycv/22-tragic.mp3",
 		"name" : "Naruto Shippuden OST - Tragic",
 		"img" : "http://image.noelshack.com/fichiers/2017/07/1487083218-naruto-sasuke-looses-arms.jpg"
+	},
+
+	{
+		"link" : "http://www.youtubeinmp3.com/download/get/?i=lhdrk5Is9Ksx9ziSHVf9e0iwzpQzUyX4&e=10&v=IK1K9O3y-gk",
```

Comment:
Doesn't return valid mp3 response, use `fetch` instead of get from `youtubeinmp3` API please